### PR TITLE
Remove unnecesary s when printing unit times

### PR DESCRIPTION
**Pull request by naferx**
Here is the logic
https://github.com/scala/scala/blob/2.11.x/src/library/scala/concurrent/duration/Duration.scala#L601-L602

### CODE REVIEW

File: akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala [hunk 1]
```diff
@@ -504,7 +504,7 @@ private[persistence] trait Eventsourced extends Snapshotter with PersistenceStas
 
       case RecoveryTick(true) ⇒
         try onRecoveryFailure(
-          new RecoveryTimedOut(s"Recovery timed out, didn't get snapshot within $timeout s"),
+          new RecoveryTimedOut(s"Recovery timed out, didn't get snapshot within $timeout"),
```
Comment (patriknw) [inline akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala]: perhaps we should use `akka.util.PrettyDuration`? I think the FiniteDuration toString is not so readable, but I might be wrong (they might have improved that)?

Comment (ktoso) [inline akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala]: So the difference is that PrettyDuration renders in the "largest, complete" time unit, for example:

```
scala> 100.seconds.toString
res9: String = 100 seconds

scala> PrettyDuration.format(100.seconds)
res10: String = 1.667 min
```

No strong feelings either way in this message about which style is better

**Comment by ktoso** [comment]
LGTM

**Comment by patriknw** [comment]
OK TO TEST

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
Thanks!

**Pull request merged**